solana-programs/anchor/audius-data/tests/test-helpers.ts

```diff
@@ -445,6 +470,141 @@ export const testUpdatePlaylist = async ({
   expect(accountPubKeys[2]).to.equal(userAuthorityKeypair.publicKey.toString());
 };
 
+export const testCreateUserDelegate = async ({
+  adminKeypair,
+  adminStorageKeypair,
+  program,
+  provider,
+  replicaSet,
+  replicaSetBumps,
+  cn1,
+  cn2,
+  cn3,
+}) => {
+  const { ethAccount, handleBytesArray, metadata, userId } =
```

Comment:
can this call into the other testCreateUser instead of duplicating the functions? i think theres a lot of duplication